src/devices/vdml_gps.cpp

```diff
@@ -57,34 +81,45 @@ double Gps::get_heading_raw() const {
 	return pros::c::gps_get_heading_raw(_port);
 }
 
-double Gps::get_rotation() const {
-	return pros::c::gps_get_rotation(_port);
```

Comment:
I understand that rotation isn't working as we thought, but is it possible to have some way to set the heading so it can at least be tared/zero'd by a hard reset? I believe that is still a valid use case.

Reply:
I think set_position() is able to do that since it takes heading as a parameter? I'll test the function again to see if it works that way.

Reply:
set_position() is able to reset the heading through set_position(get_position_x(), get_position_y(), 0)